packages/card/README.md

```diff
@@ -23,7 +23,7 @@ import Card from '@leafygreen-ui/card';
 
 <Card
   className='card-styles'
-  component='div'
+  as='div'
```

Comment:
This is the default for Box, right? Can't this be omitted, or changed to a more applicable example like "article"?

Reply:
Yes, good point!